ush/cam/ush_href_plot_py/href_atmos_util.py

```diff
@@ -933,34 +933,6 @@ def prep_prod_ghrsst_ospo_file(source_file, dest_file, date_dt):
         prepped_data.close()
```

Comment:
@MarcelCaron-NOAA - why are we making changes for href here?  Are these important href changes?

Reply:
I asked him to wriggle in some href items b/c, at the time, we didn't anticipate another href pr. I believe this is what that is. @MarcelCaron-NOAA can correct me if that is not the case.

Reply:
OK, thanks!  Would be good for some confirmation.

Reply:
Yes this is a python function that needed to be removed both because it references wget and it is not being used.  [Here is the comment on the Edit document related to this.](https://docs.google.com/document/d/1fYsVrcrjj_NxJBYG6v0sDhi-vYxShT2l2EUauHIByLs/edit?disco=AAAA4IxDgtA)  As Shelley said, it was included here because another HREF PR was not anticipated.

Reply:
OK, thanks!  If this doesn't affect href negatively, then I understand why it's here.  Thanks for all the good info.